src/pages/graphql/schema/customer/queries/is-email-available.md

```diff
@@ -4,7 +4,9 @@ title: isEmailAvailable query
 
 # isEmailAvailable query
 
-The `isEmailAvailable` query checks whether the specified email has already been used to create a customer account. A value of `true` indicates the email address is available, and the customer can use the email address to create an account.
+The `isEmailAvailable` query checks whether the specified email has already been used to create a customer account. A value of `true` indicates the email address is available, and the customer can use the email address to create an account. The value of `false` indicates the email address is not available, and the customer cannot use the email address to create an account.
```

Comment:
```suggestion
The `isEmailAvailable` query checks whether the specified email has already been used to create a customer account. As of Commerce 2.4.7, by default the query always returns a value of `true` for all email addresses. You can change this behavior by setting the **Stores > Configuration > **Sales** > **Checkout** > **Enable Guest Checkout Login** field in the Admin to **Yes**. However, doing this can expose customer information to unauthenticated users.
```